src/review/views.py

```diff
@@ -1085,20 +1085,11 @@ def add_review_assignment(request, article_id):
     :return: HttpResponse
     """
     article = get_object_or_404(submission_models.Article, pk=article_id)
-
-    # if setting enabled, fetch reviewers who have completed a review
-    # in a past review round.
-    past_reviewers = []
-    if request.journal.get_setting('general', 'display_past_reviewers'):
-        past_reviewers = logic.get_previous_round_reviewers(
-            article,
-        )
     reviewers = logic.get_reviewer_candidates(
         article,
         user=request.user,
-        reviewers_to_exclude=past_reviewers,
     )
-
+    suggested_reviewers = past_reviewers = table_sort = list()
```

Comment:
This multiple assignment will set all variables to the same one list in memory. I don't imagine that is the intended behaviour.

Perhaps it hasn't led to any problems because I don't think you are using `suggested_reviewers` or `past_reviewers` anywhere